resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.service.erb

```diff
@@ -1,3 +1,12 @@
+#
+# For example, if you want to increase <%= EZBake::Config[:project] %> open-files-limit to 10000,
```

Comment:
Maybe either remove "For example" or add some verbiage in front? like
"Local settings can be configured without being overwritten by package upgrades, for example...